pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computebackendservice/regionalcomputebackendserviceiap/dependencies.yaml

```diff
@@ -26,9 +26,12 @@ spec:
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Project
 metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: iapclient-${uniqueId}
 spec:
-  name: ComputeProjectMetadata
+  resourceID: ${TEST_DEPENDENT_ORG_PROJECT_ID}
```

Comment:
I'm assuming it's something to do with IAP ... definitely starting to see a pattern :-)